draft-irtf-cfrg-vdaf.md

```diff
@@ -5558,6 +5566,35 @@ may consider involving more than two Aggregators as described for example in
 {{star-topo}}. Note however that some schemes are not compatible with this mode
 of operation, such as Poplar1.
 
+## Defense-in-Depth Measures
+
+Prio3 and Poplar1 are designed to resist some attacks that fall outside the
+main threat model for VDAFs.
+
+Broadly speaking, domain separation is used to prevent cross protocol attacks,
+in which an attack against one VDAF translates to an attack against another.
+For example:
```

Comment:
I think the phrase "cross-protocol attack" is usually a bit stronger, in that it also covers cases where honest execution of one protocol acts as an oracle enabling an attack against another protocol. In other words, the first protocol doesn't need to have a complete attack against it in order for it to be used in a cross-protocol attack against another protocol.
```suggestion
Broadly speaking, domain separation is used to prevent cross protocol attacks,
in which data from evaluation of one VDAF translates to an attack against another.
For example:
```